docs/experiments-microblog.md

```diff
@@ -0,0 +1,44 @@
+# Anserini Experiments on TREC Microblog Collection
+
+
+### Tweets2011 collection
+
+Note that the Tweets2011 collection is distributed as a list of tweet ids that you have to download yourself, so the 
+effectiveness results you'll get should be similar, but will likely not be identical to the scores reported here.
```

Comment:
@snapbug since the code base is the same, the only parameter here is the input collection. What I mean to say is that numbers might very different depending on the collection, deletes etc. The numbers reported here are on the original collection and not after the tweets are being deleted.